lib/kill.js

```diff
@@ -47,7 +47,7 @@ export const normalizeForceKillAfterTimeout = forceKillAfterTimeout => {
 const killAfterTimeout = async ({spawned, timeout, killSignal, context, controller}) => {
 	await setTimeout(timeout, undefined, {signal: controller.signal});
 	spawned.kill(killSignal);
-	Object.assign(context, {timedOut: true, signal: killSignal});
```

Comment:
`context.signal` is not needed. When calling `spawned.kill(killSignal)`, we then wait for the process' `exit` event, which will return that signal.